server/routes/reports.ts

```diff
@@ -15,7 +15,8 @@ export default function routes({ serviceCatalogueService }: Services): Router {
     return res.render('pages/veracode')
   })
 
-  get('/veracode/data', async (req, res) => {
+  post('/veracode/data', async (req, res) => {
+    const componentsToInclude = req.body.componentNames
     const resultFilters = getResultFilters(req.query.results as string)
```

Comment:
This is a post method now, if it's the same form then they won't be in `req.query` anymore, rather `req.body` I think?

Reply:
Also shouldn't this be moved into the veracode routes file?

Reply:
Basically you have changed this to a POST from a GET but are looking in the query string for data but in [assets/js/veracode.js](https://github.com/ministryofjustice/hmpps-developer-portal/pull/228/files#diff-5c0294451916b91b895a702078c15639b36ea9b0d87cd4ce1078e366bd6aa80a) you're passing a different set of fields through the body as it's a POST.